test/cppsuite/test_harness/timestamp_manager.cxx

```diff
@@ -69,7 +69,7 @@ timestamp_manager::load()
 void
 timestamp_manager::do_work()
 {
-    std::string config;
+    std::string config, log;
```

Comment:
can we call this log_msg not log?

Reply:
75c01295dd1ce467dae2d21a80dd4aee67cd6758